config/samples/e2e/IPAM-example-IPv4/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: test
```

Comment:
Please use the default `namespace` here (not specifying a namespace at all) as for this e2e it does not matter which namespace the resources are in.